src/internal/components/drag-handle-wrapper/__tests__/drag-handle-wrapper.test.tsx

```diff
@@ -29,11 +29,28 @@ function getDirectionButton(direction: Direction) {
   );
 }
 
-function expectDirectionButtonHidden(direction: Direction) {
-  // Direction buttons get hidden via transition which doesn't end in JSDOM, so we just listen
-  // for the exiting classname instead.
-  const motionExitingClass = styles['direction-button-wrapper-motion-exiting'];
-  expect(getDirectionButton(direction)?.parentElement).toHaveClass(motionExitingClass);
+// Direction buttons get hidden via transition which doesn't end in JSDOM, so we use
+// the "exiting", "exited" or "hidden" classname instead to verify it's hidden.
+const DIRECTION_BUTTON_HIDDEN_CLASSES = [
+  styles['direction-button-wrapper-motion-exiting'],
+  styles['direction-button-wrapper-motion-exited'],
+  styles['direction-button-wrapper-hidden'],
+];
+
+function expectDirectionButtonToBeHidden(direction: Direction) {
```

Comment:
In case a direction button is active but not visible, it is rendered in the DOM to be ready for the transition - but set to invisible.

I also included the `*-motion-exiting` and `*-motion-exited` classes in that utility for more general test usage.